test/integration/xfcc_integration_test.cc

```diff
@@ -238,5 +238,683 @@ TEST_P(XfccIntegrationTest, NonTlsEnforceSanitize) {
   startTestServerWithXfccConfig("forward_only", "");
   testRequestAndResponseWithXfccHeader(makeClientConnection(), previous_xfcc_, "");
 }
+
+TEST_P(XfccIntegrationTest, TagExtractedNameGenerationTest) {
+  // Note: the test below is meant to check that default tags are being extracted correctly with
+  // real-ish input stats. If new stats are added, this test will not break because names that do
+  // not exist in the map are not checked. However, if stats are modified the below maps should be
+  // updated (or regenerated by printing in map literal format).
```

Comment:
Can you add a bit more color on how you generated this list? I'm not sure exactly what "regenerated by printing in map literal format" means. Mainly concerned that we might have to regenerate and it will be confusing.

Reply:
Sure, I can provide more details.  I could also just toss the generation code in there and leave it commented out if you think that would decrease the friction?

Reply:
+1, generally not a fan of commented out code but that seems fine here.